### DISCUSSION TITLE
Passing UMG text box as variable from blueprints to managed functions

### DISCUSSION THREAD

**Discussion by LeviLeckenby** [nxrighthere/UnrealCLR#85] [Help]
Hi, I have a 'SignIn' static method that I'm calling via the Execute Managed Function in Blue Prints, and I want to pass in the entered email and password from the UMG UI.

I can see examples where you can pass in an Actor, or SceneComponent derrived from the ObjectReference extension methods. Is it possible to pass UI objects (e.g. TextBox), or derived string variables into a static function like this?

**Comment by nxrighthere** [comment]
This API is not exposed yet, see https://github.com/nxrighthere/UnrealCLR/issues/38 and https://github.com/nxrighthere/UnrealCLR/discussions/55. Advanced blueprint nodes is also work in progress https://github.com/nxrighthere/UnrealCLR/issues/33.
